docs/examples/control_wrfchem_saveandread.yaml

```diff
@@ -0,0 +1,232 @@
+# General Description:
+# - Any key that is specific for a plot type will begin with `ts` for timeseries, `ty` for taylor.
+# - Some keys/groups are optional.
+# - For now, all plots except time series average over the analysis window.
+# - Seting axis values
+#   - If set_axis = True in data_proc section of each plot_grp,
+#     the yaxis for the plot will be set based on the values
+#     specified in the obs section for each variable.
+#   - If set_axis is set to False, then defaults will be used.
+#   - 'vmin_plot' and 'vmax_plot' are needed for
+#     'timeseries', 'spatial_overlay', and 'boxplot'.
+#   - 'vdiff_plot' is needed for 'spatial_bias' plots
+#   - 'ty_scale' is needed for 'taylor' plots.
+#   - 'nlevels' or the number of levels used in the contour plot can also optionally be provided for spatial_overlay plot.
+#   - If set_axis = True and the proper limits are not provided in the obs section,
+#     a warning will print, and the plot will be created using the default limits.
+analysis:
+  start_time: "2019-09-05-06:00:00" # UTC
+  end_time: "2019-09-06-06:00:00" # UTC
+  output_dir: ./output/save_and_read  # relative to the program using this control file
+  # ^ optional; if not specified, plots are stored in code directory (??)
+  #   Currently, the directory must exist or plot saving will error and fail.
+  debug: True
+  save:
+      paired:
+          method: 'netcdf' # 'netcdf' or 'pkl'
+          prefix: '0905' # use only with method=netcdf
+          # output_name: '0905.pkl' # use only with method=pkl
+          data: 'all'   # 'all' to save out all pairs or ['pair1','pair2',...] to save out specific pairs. With method='pkl' this is ignored and always saves all.
+      # models:
+      # obs:
+  read:
+      paired:
+          method: 'netcdf' # 'netcdf' or 'pkl'
+          filenames: {'airnow_RACM_ESRL':['./output/save_and_read/0905_airnow_RACM_ESRL.nc4'],'airnow_RACM_ESRL_VCP':['./output/save_and_read/0905_airnow_RACM_ESRL_VCP.nc4']} # example for netcdf method. Uses dict of form {group1: str or iterable of filenames, group2:...}
+          # filenames: {'airnow_wrfchem_v4.2':['0904_airnow_wrfchem_v4.2.nc4','0905_airnow_wrfchem_v4.2.nc4']} # example for netcdf method. Uses dict of form {group1: str or iterable of filenames, group2:...}
```

Comment:
Let me test whether in yaml you can split a dict across multiple lines. If you can then I'll show how to do that here. Otherwise I'll figure out a way to have the filenames split across multiple lines because that is important.

I don't think it would be hard to implement wildcards. I'll just use glob.glob() to expand each element of the list with anything returned by glob.

Reply:
I checked and it is possible to split a dict over multiple lines in the .yaml file, so no issues there.